conf/messages.en

```diff
@@ -66,7 +66,7 @@ cf.cash-account.detail.ucr=UCR
 cf.cash-account.detail.Declarationucr=Declaration UCR
 cf.cash-account.detail.declarant=Declarant
 cf.cash-account.detail.eori=EORI
-cf.cash-account.detail.importerEori=Importer EORI
+cf.cash-account.detail.importerEori=Importer <abbr title="Economic Operators Registration and Identification number">EORI</abbr>
```

Comment:
are we wanting to be putting this html inside the msg?

Reply:
yes